suite-native/link/src/components/Link.tsx

```diff
@@ -1,14 +1,60 @@
 import React from 'react';
-import { TouchableOpacity, TouchableOpacityProps } from 'react-native';
+import { GestureResponderEvent, TouchableOpacityProps } from 'react-native';
+import Animated, {
+    useAnimatedStyle,
+    interpolateColor,
+    withTiming,
+    useSharedValue,
+} from 'react-native-reanimated';
+
+import { useNativeStyles } from '@trezor/styles';
 
 import { useOpenLink } from '../useOpenLink';
 
 type LinkProps = TouchableOpacityProps & {
+    label: string;
     href: string;
 };
 
-export const Link = ({ href, ...touchableProps }: LinkProps) => {
+const ANIMATION_DURATION = 100;
+const IS_NOT_PRESSED_VALUE = 0;
+const IS_PRESSED_VALUE = 1;
+
+export const Link = ({ href, label }: LinkProps) => {
+    const { utils } = useNativeStyles();
     const openLink = useOpenLink();
+    const isPressed = useSharedValue(IS_NOT_PRESSED_VALUE);
+
+    const textColorStyle = useAnimatedStyle(() => ({
+        color: interpolateColor(
+            isPressed.value,
+            [IS_NOT_PRESSED_VALUE, IS_PRESSED_VALUE],
+            [utils.colors.textPrimaryDefault, utils.colors.textPrimaryPressed],
+        ),
+    }));
+
+    const handlePressIn = () => {
+        isPressed.value = withTiming(IS_PRESSED_VALUE, { duration: ANIMATION_DURATION });
+    };
+
+    const handlePress = (e: GestureResponderEvent) => {
+        openLink(href);
+        e.stopPropagation();
```

Comment:
Is this because of accordion items pressable handler?

Reply:
Yes.